src/components/Input/Input.stories.tsx

```diff
@@ -1,60 +1,152 @@
 import * as React from "react";
 
 import Input from "./Input";
+import { InputTypes } from "./InputTypes";
+import Label from "../Label/Label";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+import { text, boolean, select } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
 
 export default {
     title: "Input",
     component: Input,
 };
 
+let showLabel, showHelperText;
+
 export const input = () => (
-    <Input
-        labelId="blah"
-        labelContent={<>Label</>}
-        inputId="blah"
-        isRequired={false}
-    ></Input>
+    <>
+        {boolean("Show Label", true) ? (showLabel = true) : (showLabel = false)}
+        {showLabel && (
+            <Label
+                htmlFor="inputID"
+                optReqFlag={select(
+                    "Optional/Required Flag",
+                    ["Required", "Optional", ""],
+                    "Required"
+                )}
+                id={"label"}
+            >
+                {text("Input Label", "Choose your islander name: ")}
+            </Label>
+        )}
+        <Input
+            id="inputID"
+            ariaLabel="Input Label"
+            helperTextId={"helperText"}
+            required={boolean("Input Required", false)}
+            placeholder={text("Input Placeholder", "CoolPerson99")}
+            type={select("Input Type", InputTypes, InputTypes.text)}
+        ></Input>
+        {boolean("Show Helper Text", true)
+            ? (showHelperText = true)
+            : (showHelperText = false)}
+        {showHelperText && (
+            <HelperErrorText
+                isError={boolean("Helper Text Errored", false)}
```

Comment:
This knob doesn't change anything in the story either

Reply:
I'm seeing it add red styling to the HelperText to show it's error'd—funnily enough though, this is actually good UX feedback...